include/trie.h

```diff
@@ -0,0 +1,107 @@
+/*
+    A trie data structure
+*/
+
+#ifndef INCLUDE_TRIE_H_
```

Comment:
There should be an extra underscore at the start of this variable per the agreed-upon style in a prior PR.

Reply:
fixed